Sources/Puree/Output/BufferedOutput.swift

```diff
@@ -74,12 +78,13 @@ open class BufferedOutput: Output {
     }
 
     public func emit(log: LogEntry) {
-        buffer.insert(log)
+        readWriteQueue.sync {
```

Comment:
`emit` seems to be called on another DispatchQueue. 🤔 
Is this OK? 

https://github.com/cookpad/Puree-Swift/pull/11/files#diff-d4084e0da2109fc4cd8dca93448ce6efR56

Reply:
Logger's queue guarantees to call `emit` safely.
`readWriteQueue ` guarantees to access `buffer` and `logStore` safely.